test/mixer/wordlist_process.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+cat <<EOF
```

Comment:
missing copyright..
Also, please add a comment on what this shell script does.

Reply:
This is auto-generated code. I'll add one copyright on the script.